client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -33,6 +33,9 @@ export default function Dashboard(): JSX.Element {
       <Grid item className={classes.drawerWrapper}>
         <ChatSideBanner loggedInUser={loggedInUser} />
       </Grid>
+      <Grid item xs={9}>
+        <ManageBookings />
+      </Grid>
```

Comment:
I think it's better for now to create a new page for this (instead of rendering it in `Dashboard`).